### PR TITLE
Create 0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135

### PR DESCRIPTION
**Pull request by DrZee3**
ICEDOGE logo upload

### CODE REVIEW

File: assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135 [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (SarjuHansaliya) [inline assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135:1]: you need to put logo.png under `assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135`

**Review by SarjuHansaliya — CHANGES_REQUESTED** [review]

**Review by DrZee3 — COMMENTED** [review]
ICEDOGE_TOKEN_ICON_256x256

File: assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135 (hunk 1 above)
Comment (DrZee3) [reply assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135:1]: assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135/logo.png
thanks Sarju, pls check if all is good. thx

Comment (SarjuHansaliya) [reply assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135:1]: @DrZee3 you haven't added logo.png to `assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135` folder. Please go through readme and also check how other logo has been added to repo.

**Pull request closed without merging**